hi-ml-azure/tests/testazure/scripts/script_that_downloads_run_files.py

```diff
@@ -0,0 +1,30 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+import argparse
+from pathlib import Path
+
+from azureml.core import Run
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--output_path", type=str, required=True)
+
+    args = parser.parse_args()
+    output_path = Path(args.output_path)
+    output_path.mkdir(exist_ok=True)
+    print(f"Created output directory: {output_path}")
+
+    run_ctx = Run.get_context()
```

Comment:
I don't get the purpose of this script - I see it is used in a test, with compute target local. But what is being tested here, because we don't use any of our functions to download, just the API?

Reply:
Yes good catch - I should be calling our own function for download_file in this script. (The point of it it to test that download_file works inside an AML run)